content/nginxaas-azure/getting-started/ssl-tls-certificates/ssl-tls-certificates-portal.md

```diff
@@ -16,6 +16,36 @@ You can manage SSL/TSL certificates for F5 NGINXaaS for Azure (NGINXaaS) using t
 
 {{< include "/nginxaas-azure/ssl-tls-prerequisites.md" >}}
 
+## Finding the Azure Key Vault Secret Identifier {#finding-the-azure-key-vault-secret-identifier}
+
+When adding a certificate using the Azure CLI, Terraform, or ARM/Bicep templates, you need to provide the **Secret Identifier** from Azure Key Vault.
+
+{{< call-out "important" >}}
+**Use the Secret Identifier, not the Certificate Identifier.**
+
+When you create a certificate in Azure Key Vault, it creates three related objects:
+
+- A **Certificate** (with a Certificate Identifier)
+- A **Key** (for cryptographic operations)
+- A **Secret** (containing the certificate and private key as a PFX bundle)
+
+NGINXaaS requires the **Secret Identifier** to access the certificate and its private key.
+{{< /call-out >}}
+
+To find the Secret Identifier:
+
+1. Go to your Azure Key Vault in the Azure portal.
+2. Select **Certificates** from the left menu.
```

Comment:
```suggestion
1. Select **Certificates** from the left menu.
```

Use 1 on every list item here and below.  Hugo converts this to a sequence automatically